packages/osd-plugin-helpers/package.json

```diff
@@ -16,6 +16,7 @@
     "osd:watch": "tsc --watch"
```

Comment:
line 15: should change to `node ../../scripts/remove.js target && tsc` in order to bootstrap

Reply:
Thanks; fixed.